src/modules/comments/comments.controller.ts

```diff
@@ -0,0 +1,42 @@
+import "express-async-errors";
+import { Request, Response, Router } from "express";
+import { CommentsService } from "./comments.service";
+import { Comment } from "./entities/comment.entity";
+import { checkResourceOwnership } from "#/shared/middlewares/checkResourceOwnership.middleware";
+import { validatePathParameter } from "#/shared/validators/path-parameter.validator";
+import { uuidSchema } from "#/shared/schemas/uuid-schema";
+import { auth } from "#/shared/middlewares/auth.middleware";
+import { validateRequestBody } from "#/shared/validators/request-body.validator";
+import { updateCommentRequestBodyDtoSchema } from "./dto/request/update-comment-request-body.dto";
+
+export const CommentsController = Router();
+
+CommentsController.put(
```

Comment:
where are contollers for adding and retrieving comments

Reply:
fixed. 
I moved them to comments controller from blogs controller